tests/decorators/decorators.go

```diff
@@ -72,6 +72,9 @@ var (
 	RequiresRWXBlock = Label("RequiresRWXBlock")
 	// Requires a storage class with Block storage support
 	RequiresBlockStorage = Label("RequiresBlockStorage")
+	// non critical for basic kubevirt storage related functionality
+	StorageDay2 = Label("storage-day-2")
```

Comment:
How about we create storage decorators file?

Reply:
Hmm we have storage tests outside of tests/storage, so not sure. I am not that keen on having approval right here, if we ever have to, we can create one

Reply:
> Hmm we have storage tests outside of tests/storage

Isn't this a problem to be fixed? Storage maintainers should control Storage tests.

Reply:
No, some tests are cross-sig and they are not owned by sig-storage https://github.com/kubevirt/kubevirt/blob/57abd252883ea715dbb55d78a16921c7ad6b3497/tests/virtctl/memorydump.go#L53

Reply:
virtctl is a valid domain-of-its-own. Is there other SigStorage tests that should stay out of tests/storage ?

Reply:
Yep, AFAIK the clone API is maintained outside of tests/storage on purpose as well

Reply:
I'm not sure why this is so. If it is your responsibility to fix, you should have permission to edit the clone-with-storage test.

I see SigStorages in tests/vmi_configuration_test.go and test/virtiofs which I can justify even less.

Reply:
> If it is your responsibility to fix,

That's not necessarily the case. The clone API was introduced and maintained by sig-compute - 
`pkg/virt-controller/watch/clone`
(Similar point about virtiofs)

The converter bit is owned by compute today, but I totally agree that the storage bits should get extracted.